nix/sources.json

```diff
@@ -12,15 +12,15 @@
         "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
     },
     "cardano-node": {
-        "branch": "tags/1.4.0",
+        "branch": "master",
```

Comment:
commit `23c0855e3b6e2d7781931bd300c3bab4dff1365f` is now tagged as 1.5.0 if that helps.